example-get-started-experiments/code/.github/workflows/run-experiment.yml

```diff
@@ -64,32 +65,27 @@ jobs:
           STUDIO_TOKEN: ${{ secrets.STUDIO_TOKEN }}
           DVCLIVE_LOGLEVEL: DEBUG
         run: |
-          dvc pull
           cml ci --fetch-depth 0
+          dvc pull
           dvc exp run ${{ github.event.inputs.exp-run-args }}
           dvc remote add --local push_remote s3://dvc-public/remote/get-started-pools 
```

Comment:
Not really actionable, but we should try to get back to enabling [anonymous login](https://github.com/iterative/dvc/issues/5797) and then add this as secondary project remote since now it seems a bit awkward.

Reply:
Also, would there be any harm in having the push_remote already set up in the `.dvc/config`?

Reply:
Right, that's what I was thinking, but that's also awkward without anonymous login since it can't be used by anyone external. Anyway, I'm fine with it either way for now.

Reply:
I'm fine (and do this in the hacktathon repo) to have the second remote in the config. I think it's even better (can educate ppl about different remotes,. etc)